src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,100 @@
-import { useState } from 'react';
+import { useState, FC, FormEvent } from 'react';
+
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
+
+interface NewMovieProps {
+  onAdd: (Movie: Movie) => void;
+}
+
+export const NewMovie: FC<NewMovieProps> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+
+  const [newDescription, setNewDescription] = useState('');
+
+  const [newTitle, setNewTitle] = useState('');
+  const [newImgUrl, setNewImgUrl] = useState('');
+  const [newImdbUrl, setNewImdbUrl] = useState('');
+  const [newImdbId, setNewImdbId] = useState('');
```

Comment:
It's better to use a single state object for all values, simplifies state management and reduces code complexity. It also minimizes unnecessary re-renders, as updating multiple values in one object triggers only one re-render.
```suggestion
 const [formData, setFormData] = useState({
  description: '',
  title: '',
  imgUrl: '',
  imdbUrl: '',
  imdbId: ''
});
```